setup.py

```diff
@@ -50,6 +50,7 @@ def get_version(module='spyder_notebook'):
                      "to run inside Spyder as a plugin.",
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     install_requires=REQUIREMENTS,
+    include_package_data=True,
```

Comment:
Yep, tested installing with the wheel generated by `python setup.py bdist_wheel`